main.go

```diff
@@ -56,6 +57,7 @@ var (
 	shouldSetAlias              *bool
 	shouldFormat                *bool
 	shouldApplyToGeneratedFiles *bool
+	shouldSeparateNamedImports  *bool
```

Comment:
```suggestion
	shouldSeparateAliases  *bool
```

And rename everything